gdcdictionary/schemas/treatment.yaml

```diff
@@ -52,35 +52,176 @@ properties:
       - string
       - "null"
 
-  treatment_or_therapy:
+  days_to_treatment:
     term:
-      $ref: "_terms.yaml#/treatment_or_therapy"
+      $ref: "_terms.yaml#/days_to_treatment"
+    type: number
```

Comment:
Ideally yes. But it already exists in the dictionary as a number. When we go through and do the hard changes after this PR, we will probably harmonize all days_to values to integers.